includes/Admin/SetupWizard.php

```diff
@@ -12,7 +12,7 @@
 class SetupWizard {
 
     /** @var string Currenct Step */
-    protected $step = '';
+    protected string $current_step = '';
```

Comment:
Move the `$custom_log` prop declaration from  ` WeDevs\Dokan\Vendor\SetupWizard` class to  this class.

```php

    /** @var string custom logo url of the theme */
    protected $custom_logo = '';
```